components/suggest/src/suggest.udl

```diff
@@ -106,3 +106,19 @@ interface SuggestStore {
     [Throws=SuggestApiError]
     void clear();
 };
+
+interface SuggestStoreBuilder {
+    constructor();
+
+    [Self=ByArc]
+    SuggestStoreBuilder data_path(string path);
+
+    [Self=ByArc]
+    SuggestStoreBuilder cache_path(string path);
+
+    [Self=ByArc]
+    SuggestStoreBuilder remote_settings_config(RemoteSettingsConfig config);
+
+    [Throws=SuggestApiError]
+    SuggestStore finish();
```

Comment:
Nano-nit: What do you think of calling this `build()`, to match Java / Kotlin conventions?